server/api/activity/activity.model.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import mongoose from 'mongoose';
+// import {registerEvents} from './employee.events';
+
+
+const ActivitySchema = new mongoose.Schema({
+    employeeId:{
+        type:String,
+    },
+    date:{
+        type: Number
+    },
+    duration: {
+        type: String
+    },
+    activity: {
+        type: String
+    },
+    activityType: {
+        type: String
+    },
+    description: {
+        type: String
+    },
+    status: {
+        type: String
```

Comment:
should be marked as required.

Should have default status as well. Default value should be "Submit/Draft"